csrc/scheduler/mma_utils.cpp

```diff
@@ -1166,6 +1166,42 @@ inline void resolveTvToMatmulDimRolesMapping(
 
 } // anonymous namespace
 
+void scheduleTMAStoreForMmaOutput(TensorView* tv) {
+  NVF_ERROR(
+      tv->getMemoryType() == MemoryType::Global,
+      "TMA Store should write to global memory");
+
+  NVF_ERROR(
+      tv->definition()->isA<LoadStoreOp>(),
+      "This tensor should be the result of a LoadStoreOp");
+
+  NVF_ERROR(
+      tv->definition()->as<LoadStoreOp>()->opType() ==
+          LoadStoreOpType::CpAsyncBulkTensorTile,
+      "This is not a TMA operation");
+
+  NVF_ERROR(
+      tv->definition()
+              ->as<LoadStoreOp>()
+              ->in()
+              ->as<TensorView>()
+              ->getMemoryType() == MemoryType::Shared,
+      "Producer should be in shared memory");
+
+  auto mma_ops = ir_utils::getOpsOfType<MmaOp>(tv->fusion());
```

Comment:
Instead of finding it from the fusion, could we explicitly pass M and N into this function? @jacobhinkle is working on fusing multiple matmuls, so in the future, we should no longer assume there is only one matmul in the fusion.